.gitignore

```diff
@@ -77,11 +77,8 @@ web_modules/
 .yarn-integrity
 
 # dotenv environment variable files
-.env
-.env.development.local
-.env.test.local
-.env.production.local
-.env.local
+.env*
+!backend/.env.example
```

Comment:
how come we don't need to explicitly include `frontend/.env.example`?

Reply:
Whoops!